modules/soe_paragraphs/modules/soe_paragraph_cta_list/src/Plugin/paragraphs/Behavior/CtaListBehavior.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\soe_paragraph_cta_list\Plugin\paragraphs\Behavior;
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\paragraphs\Entity\ParagraphsType;
+use Drupal\paragraphs\ParagraphInterface;
+use Drupal\paragraphs\ParagraphsBehaviorBase;
+
+/**
+ * Teaser paragraph behaviors.
+ *
+ * @ParagraphsBehavior(
+ *   id = "su_cta_list_styles",
+ *   label = @Translation("CTA List Styles"),
+ *   description = @Translation("Style options for CTA list paragraph")
+ * )
+ */
+class CtaListBehavior extends ParagraphsBehaviorBase {
+
+  /**
+   * {@inheritDoc}
+   */
+  public static function isApplicable(ParagraphsType $paragraphs_type): bool {
+    return $paragraphs_type->id() == 'stanford_cta_list';
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public function defaultConfiguration(): array {
+    return [
+      'top_border' => TRUE,
+    ];
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, FormStateInterface $form_state): array {
+    $element = parent::buildBehaviorForm($paragraph, $form, $form_state);
+    $element['top_border'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Gray line above content'),
+      '#default_value' => (bool) $paragraph->getBehaviorSetting('su_cta_list_styles', 'top_border', TRUE),
+    ];
+    return $element;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function filterBehaviorFormSubmitValues(ParagraphInterface $paragraph, array &$form, FormStateInterface $form_state): array {
```

Comment:
Fun fact: the paragraph behaviors form uses NestedArray::filter(), which filters out falsey values.  In this context, the border is on by default, so it should be TRUE. But when you turn the border off, it changes to FALSE, which gets filtered out, and the default TRUE is returned. So in order to use the bool values instead of doing a simple string comparison, we have to filter the values to preserve the FALSE.  This additional function fixes that.

Reply:
You can remove this method and just reverse the logic of the checkbox. 

Make it something like "Remove Gray Line above content". Then the checkbox default value of `false` will produce the line. No need for the method

Reply:
Updated with reversed logic and removed filter